src/axom/quest/examples/quest_distributed_distance_query_example.cpp

```diff
@@ -79,6 +78,8 @@ struct Input
 
   bool randomSpacing {true};
 
+  std::vector<unsigned int> objDomainCountRange {1, 1};
```

Comment:
Out of curiosity, is there any reason not to use `axom::Array` here instead of `std::vector`?
I'm asking because if there is, that would indicate that `axom::Array` is missing some functionality.

Reply:
Simply replacing `std::vector` with `axom::Array` seems like it should work, but compile error at this line:
`circle_options->add_option("--obj-domain-count-range", objDomainCountRange)`
I couldn't immediately determine how to fix.